### PR TITLE
Add basic resleeving and SleeveMates

### PR DESCRIPTION
**Pull request by out-of-phaze**
## Description of changes
- Adds a basic form of resleeving using a body printer to print sleeves, sleevemates (or the body printer) to transfer minds into sleeves, and data disks to hold both.
- Adds a remote mainframe to hold transcore mind/body records.

To use, load a body record and optionally a mind record onto a floppy disk via a computer with a disk reader device. Then, insert that disk into the body printer and hit start. You can either AFK and let error auto-recovery slowly fix any errors in the print process that occur, or actively fix them as they crop up for it to go faster. Once it's done, eject the body.

TODO: 
- [x] Add server room
- [x] Add TransCore mainframe
- [x] Add mind records
- [x] Add body records
- [x] Add body printer
- [x] Add sleevemate
- [x] Add sleeve transfer
- [x] Test locally
- [ ] Make the transcore mainframe automount on the resleeving computer.
- [ ] Make the body printer use 'biomass' instead of bone/meat materials. It's just super hard to get bone in any meaningful amount.

TODO in a future PR:
- [ ] Add backup implants
- [ ] Add sleeve designer console

## Why and what will this PR improve
Necessary for CitRP parity.

## Authorship
Mazian on Hearth of Hestia for the body printer, CitRP/Virgo coders for the sleevemate, me (Penny) for heavy adaptation to make them work on Nebula.

## Changelog
:cl:
add: Added basic resleeving
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/circuitboards/machinery/network.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		/obj/item/stock_parts/power/apc/buildable = 1,
 		/obj/item/stock_parts/console_screen = 1,
 		/obj/item/stock_parts/keyboard = 1,
-		/obj/item/stock_parts/computer/hard_drive/cluster/fullhouse
+		/obj/item/stock_parts/computer/hard_drive/cluster/empty = 1
```
Comment (Zandario) [inline code/game/objects/items/weapons/circuitboards/machinery/network.dm:11]: Trailing commas :(

File: mods/content/resleeving/body_designer.dm
Comment (Zandario) [inline mods/content/resleeving/body_designer.dm:1]: Including a empty file for a reason?

**Review by Zandario — APPROVED** [review]
I don't see any criminal behavior

File: mods/content/resleeving/body_designer.dm
Comment (out-of-phaze) [reply mods/content/resleeving/body_designer.dm:1]: ahhh shit. I mean sure I guess, it's a good reminder to add it later, but it's not necessary in the first version

**Pull request merged**